tests/interactive/test_app.py

```diff
@@ -69,12 +70,13 @@ async def test_input():
                 res = arith.Muli(n, two)
                 func.Return(res)
 
+        await pilot.pause()
```

Comment:
why is this necessary? we're not simulating any user actions between this and the previous pause, I'd expect the test to work without it